test/examples/uct_hello_world.c

```diff
@@ -200,34 +220,40 @@ ucs_status_t do_am_zcopy(iface_info_t *if_info, uct_ep_h ep, uint8_t id,
     return status;
 }
 static void print_strings(const char *label, const char *local_str,
-                          const char *remote_str)
+                          const char *remote_str, size_t length)
 {
-    fprintf(stdout, "\n\n----- UCT TEST SUCCESS ----\n\n");
-    fprintf(stdout, "[%s] %s sent %s", label, local_str, remote_str);
-    fprintf(stdout, "\n\n---------------------------\n");
-    fflush(stdout);
+    char *str = calloc(1, length);
+    if (str != NULL) {
+        mem_type_allocators[test_mem_type].memcpy(str, remote_str, length);
+        fprintf(stdout, "\n\n----- UCT TEST SUCCESS ----\n\n");
+        fprintf(stdout, "[%s] %s sent %s", label, local_str, str);
+        fprintf(stdout, "\n\n---------------------------\n");
+        fflush(stdout);
+        free(str);
+    }
```

Comment:
Same as above, should there be something happening if allocation of `str` fails?